swarm.cabal

```diff
@@ -38,7 +38,7 @@ maintainer: byorgey@gmail.com
 bug-reports: https://github.com/swarm-game/swarm/issues
 copyright: Brent Yorgey 2021
 category: Game
-tested-with: ghc ==9.2.8 || ==9.4.8 || ==9.6.5 || ==9.8.2
+tested-with: ghc ==9.2.8 || ==9.4.8 || ==9.6.5 || ==9.8.2 || ==9.10.1
```

Comment:
Following up with https://github.com/swarm-game/swarm/pull/1996
We can probably drop GHC <= 9.4

Reply:
I think we should wait with GHC 9.4 since GHCup recommends it. But we could drop 9.2.